theme/404.html

```diff
@@ -109,18 +143,29 @@ <h1 class="no-auto-render">
                     </div>
                 </div>
             </footer>
+            <script src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.4/lodash.js"></script>
```

Comment:
We should arguably be hosting lodash ourselves, but given that I copied this line from `base.html`, that's a question for a different PR.